src/store/tools/useAnnotationTool.ts

```diff
@@ -1,34 +1,44 @@
 import { Ref, computed, ref } from 'vue';
 import { Store } from 'pinia';
-import { removeFromArray } from '@/src/utils';
 
+import { PartialWithRequired } from '@/src/types';
 import { TOOL_COLORS } from '@/src/config';
+import { removeFromArray } from '@/src/utils';
 import { useCurrentImage } from '@/src/composables/useCurrentImage';
-import {
-  FrameOfReference,
-  frameOfReferenceToImageSliceAndAxis,
-} from '@/src/utils/frameOfReference';
+import { frameOfReferenceToImageSliceAndAxis } from '@/src/utils/frameOfReference';
 import { useViewStore } from '@/src/store/views';
 import { getLPSAxisFromDir } from '@/src/utils/lps';
 import { LPSAxisDir } from '@/src/types/lps';
+import { AnnotationTool } from '@/src/types/annotationTool';
 import { findImageID, getDataID } from '@/src/store/datasets';
-import useViewSliceStore from '../view-configs/slicing';
 
-type AnnotationTool = {
-  id: string;
-  color: typeof TOOL_COLORS[number];
-  imageID: string;
-  slice: number;
-  placing?: boolean;
-  frameOfReference: FrameOfReference;
+import useViewSliceStore from '../view-configs/slicing';
+import { useLabels, Labels } from './useLabels';
+
+const annotationToolDefaults = {
+  frameOfReference: {
+    planeOrigin: [0, 0, 0],
+    planeNormal: [1, 0, 0],
+  },
+  slice: -1,
+  imageID: '',
+  placing: false,
+  color: TOOL_COLORS[0],
+  name: 'baseAnnotationTool',
```

Comment:
Should we have a default name? Or just leave it empty?

Reply:
no opinion.  Was just following ruler which has 'Ruler" as default name.